guides/msearch.md

```diff
@@ -8,7 +8,7 @@ OpenSearch's Multi-Search (`msearch`) API allows you to execute multiple search
 const host = "localhost";
 const protocol = "https";
 const port = 9200;
-const auth = "admin:admin";
```

Comment:
This code block is supposed to be able to executed as is. This change will cause an error to throw right?

Reply:
Same for similar changes in the `./guides` folder.

Reply:
Yes most likely it will cause an error, but we are intending users to set admin passwords now. I see a similar behavior in the line below:
```
const ca_certs_path = "/full/path/to/root-ca.pem";
```
Which seems to expect users to change this area?

Reply:
Can we change it to `admin:myStrongPassword123!` since you already have that set in the dockerfile? The goal is lower the barrier of entry for the users so that they have to make as few changes as possible :)

Reply:
Done in: 47b573e